CMakeLists.txt

```diff
@@ -217,9 +217,6 @@ add_compile_options(
     $<$<COMPILE_LANG_AND_ID:C,Clang>:-Wno-extra-semi-stmt>
     $<$<COMPILE_LANG_AND_ID:C,Clang>:-Wno-missing-noreturn>
     $<$<COMPILE_LANG_AND_ID:C,Clang>:-Wno-cast-align>
-
-    # Suppressions required to build clean with GCC.
-    $<$<COMPILE_LANG_AND_ID:C,GNU>:-Wno-declaration-after-statement>
```

Comment:
If we are removing this have you checked build with gcc

Reply:
Yes, the C90 compliance checks has been done on GCC and Clang